src/components/modecules/ConfirmedCasesSummary.js

```diff
@@ -49,6 +49,7 @@ const ConfirmedCasesSummary = props => {
             style={{
               color: mapColorForStatus(v.fieldValue).main,
               fontWeight: 600,
+              marginRight: "0.5rem",
             }}
           >
```

Comment:
or what's in my mind:
```suggestion
          >
          {i < Object.keys(status).length - 1? " | " : ""}
```